template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/RazorPagesWeb-CSharp/Pages/Error.cshtml

```diff
@@ -13,3 +14,9 @@
 <p>
     <strong>Development environment should not be enabled in deployed applications</strong>, as it can result in sensitive information from exceptions being displayed to end users. For local debugging, development environment can be enabled by setting the <strong>ASPNETCORE_ENVIRONMENT</strong> environment variable to <strong>Development</strong>, and restarting the application.
 </p>
+@if (Activity.Current != null)
```

Comment:
Can we move this into a page model please?

**Error.cshtml**
``` html
@page
@model ErrorModel
@{
    ViewData["Title"] = "Error";
}
...
@if (Model.ShowActivityId)
{
    <p>
        <strong>Correlation ID:</strong> <code>@Model.CorrelationId</code>
    </p>
}
```

**Error.cshtml.cs**
``` c#
public class ErrorModel : PageModel
{
    public string CorrelationId { get; set; }

    public bool ShowCorrelationId => !string.IsNullOrEmpty(CorrelationId);

    public void OnGet()
    {
        CorrelationId = Activity.Current?.Id;
    }
}
```